### PR TITLE
Update primary mixin.

### PR DESCRIPTION
**Pull request by notlee**
And update migration guide. And remove outdated design guidelines.

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-teaser bundle size difference from 3.5.9 to b26c0bffc4174a1f7b06a1aa99abae8ff7dddb38
No significant bundle size differences found.

File: MIGRATION.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+## Migration guide
```
Comment (chee) [inline MIGRATION.md]:
```suggestion
# Migration guide
```

File: README.md [hunk 2]
```diff
@@ -15,7 +14,7 @@ Install the [Origami build tools](https://github.com/Financial-Times/origami-bui
 
 ## Usage
```
Comment (chee) [inline README.md]: i think you probably wanted to remove this heading ^_^

**Review by chee — APPROVED** [review]
couple of notes, but:

File: README.md (hunk 2 above)
Comment (notlee) [reply README.md]: I did!

**Comment by origamiserviceuser** [comment]
o-teaser bundle size difference from 3.5.9 to 774ba027909e1945490e543866af9f8f31b84c13
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-teaser bundle size difference from 3.5.9 to dcdb01d52cf5ea4d28f2e4c115a84e29de44819c
No significant bundle size differences found.

**Pull request merged**